libraries/AP_L1_Control/AP_L1_Control.cpp

```diff
@@ -64,7 +64,7 @@ float AP_L1_Control::get_yaw()
 /*
   Wrap AHRS yaw sensor if in reverse - centi-degress
  */
-float AP_L1_Control::get_yaw_sensor()
+int32_t AP_L1_Control::get_yaw_sensor()
```

Comment:
This is arguably worse as you are doing more casting now then before (on both the return and the one spot this is used).

Reply:
really?  I don't see why because the values used within get_yaw_sensor are all int32_t.. and the one place where it's used it also wants an int32..

Reply:
Perhaps the issue is that someone has messed up with the wrap_180_cd so it always returns a float now.  It use to return int32_t..

Reply:
To keep things moving, I've removed the L1 change.